src/lib/io/include/sol-spi.h

```diff
@@ -20,8 +20,10 @@
 
 #include <stdint.h>
 #include <stdbool.h>
+#include <sys/types.h>
```

Comment:
@cabelitos I remember seeing compilation error _‘ssize_t’ has not been declared_ when sol-spi.h was included in the bindings, so I added this. Now, I tried with the latest sources and I don't see the error even after removing this include, so we probably don't need it.